pkg/query/storeset_test.go

```diff
@@ -450,7 +453,7 @@ func TestStoreSet_Update(t *testing.T) {
 	discoveredStoreAddr = append(discoveredStoreAddr, stores2.StoreAddresses()...)
 
 	// New stores should be loaded.
-	storeSet.Update(context.Background())
+	storeSet.Update(context.Background(), false)
```

Comment:
Generally a good rule is to avoid Boolean variables in function. Especially in this case it might be better to just have different function... WDYT?

Reply:
I agree that an enum such as `QuerierEvictionPolicy` with values like `StrictMode` and `LaxMode` would make things cleaner here. However, you've mentioned that you'd like to see another function for this. Could you elaborate on that?

Reply:
@bwplotka ping.

Reply:
@bwplotka ping.